README.md

```diff
@@ -90,6 +90,22 @@ You are now ready to start a local development server on http://localhost:3000 v
 npm run dev
 ```
 
+## Development
+
+### TypeScript
+
+Since this app uses TypeScript and the data ist stored in a PostgreSQL database, we need a way to ensure type-safety for the requested data. We do this by fetching the types from the database schema and converting them to TypeScript types, to be found in `./src/lib/types/database.ts`.
```

Comment:
Maybe extend this with this text:
```
This works because we use [postgrest-js](https://github.com/supabase/postgrest-js) which is developed by [Supabase](https://supabase.com/) so all the type definitions created with the [supabase/cli](https://github.com/supabase/cli) are easily insertable into the postgrest-js client.
```